build/yamls/antrea-windows-ovs-containerd.yml

```diff
@@ -0,0 +1,336 @@
+apiVersion: v1
+data:
+  Install-WindowsCNI-Containerd.ps1: |
+    $ErrorActionPreference = "Stop";
+    mkdir -force c:/var/log/antrea
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+    mkdir -force c:/opt/cni/bin/
+    mkdir -force c:/etc/cni/net.d/
+    cp $mountPath/k/antrea/cni/* c:/opt/cni/bin/
+    cp $mountPath/etc/antrea/antrea-cni.conflist c:/etc/cni/net.d/10-antrea.conflist
+    mkdir -force c:/k/antrea/bin
+    cp $mountPath/k/antrea/bin/antctl.exe c:/k/antrea/bin/antctl.exe
+  Run-AntreaAgent-OVS-Containerd.ps1: |
+    $ErrorActionPreference = "Stop"
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+    Start-Sleep -Seconds 3
+    $env:PATH = [Environment]::GetEnvironmentVariable("PATH", "User")
+    & "$mountPath/k/antrea/bin/antrea-agent.exe" --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0
+  Run-AntreaOVS-Containerd.ps1: |
+    $ErrorActionPreference = "Stop"
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
+    $newPath = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+    [Environment]::SetEnvironmentVariable("PATH", $newPath, "User")
+    $env:PATH = [Environment]::GetEnvironmentVariable("PATH", "User")
+    $OVS_DB_SCHEMA_PATH = "$mountPath/openvswitch/usr/share/openvswitch/vswitch.ovsschema"
+    $OVSInstallDir = "C:\openvswitch"
+    $OVS_DB_PATH = "$OVSInstallDir\etc\openvswitch\conf.db"
+    if ($(Test-Path $OVS_DB_SCHEMA_PATH) -and !$(Test-Path $OVS_DB_PATH)) {
+        ovsdb-tool create "$OVS_DB_PATH" "$OVS_DB_SCHEMA_PATH"
+    }
+    ovsdb-server $OVS_DB_PATH -vfile:info --remote=punix:db.sock --log-file=/var/log/antrea/openvswitch/ovsdb-server.log --pidfile --detach
+    ovs-vsctl --no-wait init
```

Comment:
Is this required? I don't know we call init in other cases.

Reply:
The ovs-vsctl --no-wait init command is not mandatory in all cases, but it is often recommended to ensure proper initialisation of the OVS configuration database. For first time it is fine,
but one risk i see  is that it might clear any existing configurations in the OVS database. @Atish-iaf

Reply:
From ovs documentation

> Initialize the database using ovs-vsctl. This is only necessary the first time after you create the database with ovsdb-tool, though running it at any time is harmless: